projects/backend/fronts.ts

```diff
@@ -60,6 +61,22 @@ export const parseCollection = async (
                 (furniture && furniture.kicker) || article.kicker || '' // I'm not sure where else we should check for a kicker
             const headline =
                 (furniture && furniture.headlineOverride) || article.headline
+            const trail =
```

Comment:
Not long now https://github.com/microsoft/TypeScript/issues/16#issuecomment-515160784